Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,328 @@
+
+![Grafana home page](media/ceph-dashboard-host-monitoring.png)
+
+_Note: Until node exporter will be installed metrics about nodes will not be available (see alert manager section in this document)_ 
+
+## Prometheus Node exporter
+
+Metrics coming from the cluster nodes are recollected by Prometheus Node exporter, this application must be installed in all the cluster nodes and communicate properly with Prometheus server.
+
+Execute:
+```console
+kubectl create -f node-exporter.yaml
+```
+
+After that, you must have a new daemonset with the related pods running in your kubernetes cluster
+
+```console
+kubectl create -f node-exporter.yaml
+```
+>```
+>daemonset.apps/node-exporter created
+>```
+```console
+kubectl get daemonset -n rook-ceph
+```
+>```
+>NAME               DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+>...
+>node-exporter      3         3         3       3            3           <none>          17s
+>```
+```console
+kubectl get pod -l app.kubernetes.io/name=node-exporter -n rook-ceph
+```
+>```
+>NAME                  READY   STATUS    RESTARTS   AGE
+>node-exporter-4x4g2   1/1     Running   0          3m49s
+>node-exporter-w6gz8   1/1     Running   0          3m49s
+>node-exporter-xzfb5   1/1     Running   0          3m49s
+```
+
+Next step is to instruct the Prometheus server about how to scrape metrics from these pods, execute:
+
+```console
+kubectl create secret generic additional-scrape-configs --from-file=prometheus-additional.yaml -oyaml -n rook-ceph > additional-scrape-configs.yaml
+kubectl create -f additional-scrape-configs.yaml
+kubectl patch prometheus rook-prometheus -n rook-ceph --type merge --patch "$(cat scrape-config-patch.yaml)
+```
+
+Finally, just verify that the metrics coming from the cluster nodes are present in the Prometheus server.
+
+```console
+echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900/targets"
+```
+>```
+>http://192.168.122.70:30900/targets
+>```
+
+Type the URL returned in your internet browser, you should see something like:
+
+![Grafana home page](media/prometheus_targets.png)
+
+## Prometheus alert manager
+
+In order to enable the complete set of alerting features provided by the Ceph dashboard, a Prometheus Alert Manager must be accessible.
+
+Execute:
+```console
+kubectl create -f alert-manager.yaml
+```
+>```
+>configmap/alertmanager-config created
+>service/alertmanager created
+>deployment.apps/alertmanager created
+```
+
+You should be able now to connect with the alert manager using the following URL:
+
+```console
+echo "http://$(kubectl get pod -l app=alertmanager -n rook-ceph -o jsonpath={.items[0].status.hostIP}):$(kubectl get service alertmanager -n rook-ceph  -o jsonpath='{.spec.ports[0].nodePort}')"
```

Comment:
![Screenshot from 2021-10-04 12-06-52](https://user-images.githubusercontent.com/34444611/135804698-aaef1d61-e678-4d29-9fe0-5102dce6584d.png)

Reply:
when I opened the file two places are not rendering correctly with quotes. I have attached the screenshots.

FYI, in VS code there are extensions to review markdown files